bulk.yml

```diff
@@ -0,0 +1,13 @@
+---
+continue: "true"
+transfer: "ln_s"
+exclude: "clientpath"
+checksum_algorithm: "File-Size-64"
+logprefix: "logs/"
+output: "yaml"
+# Default columns for the regular screens.
+# This may need to be modified in other bulk files.
+columns:
```

Comment:
The `columns` in `idr0040-experimentA-bulk.yml` could be fully copied here and removed from the other file for further simplification. Overall :+1: